src/Resources.mdx

```diff
@@ -0,0 +1 @@
+# Resources
```

Comment:
- [ ] Let's archive this page for now. (OK to `git rm`, but also remove it from the `.doczrc` file if needed)